cheng-k/2025-Summer.md

```diff
@@ -6,32 +6,83 @@
 
-#### 🌵   
+#### 🌵
 
-* 
+* The "The Turing Way" book is written with great depth. It is sometimes difficult to recognize what information can be added simply due to the fact that it is already so detailed. Some possible additions are already written elsewhere in the book, so I am not sure if it is a matter of linking to it or adding a copy.
 
 ### 🤝 Reciprocal support
 
 #### Who did I help?
 
-* 
+* Trying to identify ways to (further) improve The Turing Way book
 
 #### Who helped me?
 
-* 
+* KW
+  * Introduced me to BIDS and laid foundation for next few weeks
+  * Gave tour of AI Futures space
+  * Provided background and motivation for BIDS, open source, and open science.
+  * Suggested areas in The Turing Way book that could be improved and explained how to navigate the repository/book
+  * Assigned resources (videos, existing written documentation) to consult
+  * Create github issue on The Turing Way repository
+* JK
+  * Granted cory building keycard access and tour
+  * Contacted IT department as I was having difficulty logging into my Google account (resolved)
+  * Intro to being an intern here
+* LWH
+  * BIDS suite: keycard access, tour, and desk assignment
+  * Sent me suite welcome email, requested my title+headshot+bio (todo)
 
 ### Activities / tasks
 
 #### What did I do?
 
-* 
+##### Main
+
+* Watched [Stefan's talk](https://www.youtube.com/watch?v=cmUtiEv4xm4) from last year and took notes (3 hours)
+  * Good intro to the community behind open science, can elaborate more from this video if needed.
+  * I am really surprised that 50% of UC Berkeley undergraduates take Data8 - Intro to Data Science!
+* Read through The Turing Way book, mainly these sections (6 hours)
+  * [Guide for Reproducible Research](https://book.the-turing-way.org/reproducible-research/reproducible-research)
+  * [Communication Guide](https://book.the-turing-way.org/communication/comms-overview)
+* Watch 3 three [AIM-rsf](https://github.com/aim-rsf/training/tree/main/version-control) videos and read slides (4 hours)
+  * "intro to git", "intro to github", "github for collaboration"
+* Compare these two sources of information. Identify areas that need changing, improving, or adding. Still in progress. (8 hours)
+  * I wrote up these suggested changes in another doc, will share soon.
+  * I also found a few typos so maybe this would be a nice first PR to the repository.
+* Read through some past PRs and existing issues on The Turing Way to understand how to contribute (1 hour)
+* Forked The Turing Way repository and set up a local edit enviroment, and installed all necessary libraries and software. Learned about Jupyter Books (1 hour)
+* Wrote up this report document (2 hours)
+
+##### Misc
```

Comment:
I just realised - have you counted the hours that we spent together? When we went down to WeWork etc? Which category are those in?